test-app/gradle.properties

```diff
@@ -0,0 +1,16 @@
+mlHost=localhost
+# Define these in gradle-local.properties
+mlUsername=
+mlPassword=
+
+mlStagingForestsPerHost=1
+mlFinalForestsPerHost=1
```

Comment:
Bump this and staging to 2 - just to make sure we don't introduce any bugs that don't appear when there's only one forest. Jobs can remain at 1 forest, as we don't care about jobs data at all.